app/src/main/kotlin/com/twain/interprep/presentation/ui/theme/Color.kt

```diff
@@ -5,4 +5,18 @@ import androidx.compose.ui.graphics.Color
 val Purple200 = Color(0xFFBB86FC)
 val Purple500 = Color(0xFF6200EE)
 val Purple700 = Color(0xFF3700B3)
-val Teal200 = Color(0xFF03DAC5)
\ No newline at end of file
+val Teal200 = Color(0xFF03DAC5)
+
+val Primary30 = Color(0xFF4F378B)
+val LightPurple = Color(103, 80, 164, 64)
+val Green = Color(0xFF008B00)
+val LightGreen = Color(144, 238, 2, 64)
+val DarkGray = Color(0xFF3F3D56)
+val Gray = Color(97, 97, 97, 64)
+val DateTimeGray = Color(0xFF625B71)
+val BrightGreen = Color(0xFF09AF00)
```

Comment:
I understand that the name of those color doesn't make sense, but I don't know which naming format should I follow

Reply:
I have updated. Plz follow the same.